cmd/skopeo/sync.go

```diff
@@ -477,7 +513,24 @@ func imagesToCopy(source string, transport string, sourceCtx *types.SystemContex
 			return descriptors, errors.Errorf("No images to sync found in %q", source)
 		}
 		descriptors = append(descriptors, desc)
+	case oci.Transport.Name():
+		desc := repoDescriptor{
+			Context: sourceCtx,
+		}
 
+		if _, err := os.Stat(source); err != nil {
+			return descriptors, errors.Wrap(err, "Invalid source directory specified")
+		}
+		desc.DirBasePath = filepath.Clean(source)
```

Comment:
`dirBasePath` is only documented to apply to `dir:` — and AFAICS isn’t used in the PR as is anyway.